migration_scripts/sentiment_twitter.py

```diff
@@ -0,0 +1,50 @@
+"""Generic batch script for data migration on all three data sources
+If the field sentiment doesn't exist, add a sentiment analysis of the field text to the article, post or tweet.
+! The same script can be used for all three data sources, as only line 11 to 13 changes !
+"""
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+from utils import read_args, get_database
+
+if __name__ == '__main__':
+    remote = read_args(__file__)
+    database = get_database(remote)
+    # collection = database['rss.articles']
```

Comment:
maybe give the collection name as an argument, so we don't have to touch the code at all when executing the scripts on the server?

Reply:
Good idea, is done. Also the limit of modifying entries is a commandline argument now.